ThunderCloud/NavigationController.swift

```diff
@@ -11,6 +11,7 @@ import SafariServices
 import MessageUI
 import StoreKit
 import ThunderTable
+import AVKit
```

Comment:
Is this necessary?

Reply:
Yep, used on line 312 to create an `AVPlayer` :)